types/ibmcloud-appid/index.d.ts

```diff
@@ -0,0 +1,75 @@
+// Type definitions for ibmcloud-appid 6.2
+// Project: https://github.com/ibm-cloud-security/appid-serversdk-nodejs
+// Definitions by: Younes A <https://github.com/younes-io>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ If this module is a UMD module that exposes a global variable 'myLib' when
```

Comment:
```suggestion
/*~ If this module is a UMD module that exposes a global variable 'myLib' when
```
please remove this block and any other block which are obvious instructional commands from template. Add your own or leave without comments

Reply:
Done